fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ReviewQualityCheckCards.kt

```diff
@@ -64,6 +66,7 @@ fun ReviewQualityCheckExpandableCard(
                 .fillMaxWidth()
                 .clickable {
                     isExpanded = isExpanded.not()
+                    onToggleExpandedState(isExpanded)
```

Comment:
This is fine for now but we should revisit this to move isExpanded for settings and info to the state, so the logic for expanded if check would be moved to the telemetry middleware.